R/plot.R

```diff
@@ -780,3 +780,519 @@ make_waterfall_plot <- function(dt_plot,
 
   return(gg)
 }
+
+
+#' Plots of the MSEv Evaluation Criterion
+#'
+#' @description
+#' Make plots to visualize and compare the MSEv evaluation criterion for a list of
+#' [shapr::explain()] objects applied to the same data and model. The function creates
+#' bar plots and line plots with points to illustrate the overall MSEv evaluation
+#' criterion, but also for each observation/explicand and combination by only averaging over
+#' the combinations and observations/explicands, respectively.
+#'
+#' @inheritParams plot.shapr
+#' @inheritParams default_doc
+#'
+#' @param explanation_list A list of [shapr::explain()] objects applied to the same data and model.
+#' If the entries in the list are named, then the function use these names. Otherwise, they default to
+#' the approach names (with integer suffix for duplicates) for the explanation objects in `explanation_list`.
+#' @param id_combination Integer vector. Which of the combinations (coalitions) to plot.
+#' E.g. if you used `n_combinations = 16` in [explain()], you can generate a plot for the
+#' first 5 combinations and the 10th by setting `index_x_explain = c(1:5, 10)`.
+#' @param geom_col_width Numeric. Bar width. By default, set to 90% of the [ggplot2::resolution()] of the data.
+#' @param N_sd_error_bars Positive numeric (default is `1`). The number of standard deviation to scale error bars in
```

Comment:
Mention that 0 removes it.

Reply:
Specified that setting `levels = NULL` in the new version removes the CI.